qds_sdk/commands.py

```diff
@@ -977,6 +977,10 @@ class DbExportCommand(Command):
                          help="Modes 1 and 2: DbTap Id of the target database in Qubole")
     optparser.add_option("--db_table", dest="db_table",
                          help="Modes 1 and 2: Table to export to in the target database")
+    optparser.add_option("--use_customer_cluster", dest="use_customer_cluster", default=False,
```

Comment:
@NamrataAShettar : Can you explain the api behaviour for the following:

1. use_customer_cluster is True but customer_cluster_label is not set. 
2. customer_cluster_label is set but use_customer_cluster is False

Should we perform these checks here instead of sending those requests to server?

Also, add tests covering these cases.